qfieldsync/gui/package_dialog.py

```diff
@@ -168,6 +173,14 @@ def package_project(self):
         self.qfield_preferences.set_value("exportDirectoryProject", export_folder)
         self.dirsToCopyWidget.save_settings()
 
+        tmp_path = os.path.dirname(self.tmp_project)
```

Comment:
```suggestion
        tmp_project_filepath = Path(self.tmp_project_filename).parent
```

Or whatever is the syntax using `pathlib`. Note that in all the codebase we prefer using `pathlib` over `os.path`.